aviatrix/resource_aviatrix_account.go

```diff
@@ -733,13 +733,11 @@ func resourceAviatrixAccountCreate(ctx context.Context, d *schema.ResourceData,
 		return diag.Errorf("failed to create Aviatrix Account: %s", err)
```

Comment:
Can we simplify this? Its the same error returned twice. Can we just check if the err is nil before doing casting? Then return the casted error if the error matches? Just confusing logic.

Something like `if err != nil { if _, ok := err.(...) {`

Reply:
Fixed, but also realized something else in the process.  I need to test this change a bit more thoroughly.